doc/version_1/040_treesForVulMgmt.md

```diff
@@ -56,7 +56,7 @@ Table 3: Proposed Meaning for Deployer Priority Outcomes
 | ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Defer            | Do not act at present.                                                                                                                          |
 | Scheduled        | Act during regularly scheduled maintenance time.                                                                                                   |
-| Out-of-cycle     | Act more quickly than usual to apply the fix out-of-cycle, during the next available opportunity, working overtime if necessary.                    |
+| Out-of-cycle     | Act more quickly than usual to apply the remediation out-of-cycle, during the next available opportunity, working overtime if necessary.                    |
```

Comment:
Suggest s/apply the remediation/act/ here. I don't think this definition should prefer remediation over mitigation as the action the deployer might take.